docker/mingw.sh

```diff
@@ -95,7 +97,7 @@ EOF
 -targets := i686-w64-mingw32 x86_64-w64-mingw32
 +targets := i686-w64-mingw32
 -threads := posix win32
-+threads := posix
++threads := posix win32
```

Comment:
Seems like this patch line isn't needed anymore since `-` and `+` are the same?

Reply:
My bad. Fixed now.